Lombiq.HelpfulExtensions.Tests.UI/Extensions/TestCaseUITestContextExtensions.cs

```diff
@@ -135,6 +136,57 @@ public static async Task TestFeatureCodeGenerationsAsync(this UITestContext cont
         });
     }
 
+    /// <summary>
+    /// Tests the Lombiq Helpful Extensions - Content Sets feature.
+    /// </summary>
+    public static async Task TestFeatureContentSetsAsync(this UITestContext context)
+    {
+        const string contentId0 = "contentsetexample000000000";
```

Comment:
We could extract these and move them into the Constants folder.

Reply:
There isn't a point to it, because all the test extension methods will always be in this TestCaseUITestContextExtensions.cs file.